mkdocs/docs/getting-started/concepts.md

```diff
@@ -4,21 +4,20 @@ template: main.html
 
 # What is Iter8?
 
-Kubernetes release engineering toolkit built for DevOps, MLOps, SRE and data science teams. 
+Kubernetes release optimizer built for DevOps and MLOps teams.
 
 ## What is an Iter8 experiment?
-Iter8 experiments make it simple to collect performance and business metrics for Kubernetes apps and ML models, assess and compare multiple app/ML model versions, safely rollout winning versions, and maximize business value with each release.
+Iter8 experiments make it simple to collect performance and business metrics for apps and ML models, assess, compare and validate multiple app/ML model versions, safely rollout winning version, and maximize business value in each release.
```

Comment:
"safely rollout winning version" should be "safely rollout **a** winning version"

Reply:
Done.